examples/with-react-native-web/next.config.js

```diff
@@ -1,10 +1,13 @@
 module.exports = {
-  webpack: config => {
+  webpack: (config, { defaultLoaders }) => {
     config.resolve.alias = {
       ...(config.resolve.alias || {}),
       // Transform all direct `react-native` imports to `react-native-web`
       'react-native$': 'react-native-web'
```

Comment:
Potentially if you're importing something from a subfolder of `react-native` eg `import something from 'react-native/a/b/c` this value is wrong as `$` would make it match only `import something from 'react-native'` and not subfolders.

So potentially changing it to:

```suggestion
      'react-native': 'react-native-web'
```

Will solve the issue.

Reply:
react-native-web doesn't have the same file structure as react-native, it would be better if relative imports were disabled.